app/controllers/providers/client_received_legal_helps_controller.rb

```diff
@@ -0,0 +1,36 @@
+module Providers
+  class ClientReceivedLegalHelpsController < BaseController
+    include ApplicationDependable
+    include Steppable
+    include SaveAsDraftable
+
+    def show
+      @form = MeritsAssessments::ClientReceivedLegalHelpForm.new(model: merits_assessment)
```

Comment:
why have you not restrict this application, to be view by this provider only?

Reply:
I forgot. It's done now :)

Reply:
👍